cd/prow/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Container image for running ACK CD prow jobs.
+FROM quay.io/containers/buildah
```

Comment:
Do they have an LTS build tag we can cache?

Reply:
I will use one of the standard release rather than latest.